File: ed1/gara4/strudel/att/strudel.c
#include <stdio.h>
#include <assert.h>

#define MAXN 100000

int porziona(int N, int mandorle[], int cannella[]) {
    // Mettete qui il codice della soluzione
    return 42;
}


int mandorle[MAXN], cannella[MAXN];

int main() {
    FILE *fr, *fw;
    int N, i;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(1 == fscanf(fr, "%d", &N));
    for(i=0; i<N; i++)
        assert(1 == fscanf(fr, "%d", &mandorle[i]));
    for(i=0; i<N; i++)
        assert(1 == fscanf(fr, "%d", &cannella[i]));

    fprintf(fw, "%d\n", porziona(N, mandorle, cannella));
    fclose(fr);
    fclose(fw);
    return 0;
}

File: ed1/gara4/strudel/att/strudel.pas
function porziona(N: longint; var mandorle, cannella: array of longint): longint;
begin
    (* Mettete qui il codice della soluzione *)
    porziona := 42;
end;


const
    MAXN = 100000;
var
    N, i                : longint;
    mandorle, cannella  : array[0..MAXN-1] of longint;
    fr, fw              : text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N);
    for i:=0 to N-1 do
        read(fr, mandorle[i]);
    readln(fr);
    for i:=0 to N-1 do
        read(fr, cannella[i]);

    writeln(fw, porziona(N, mandorle, cannella));
    close(fr);
    close(fw);
end.
